hudi-utilities/src/main/java/org/apache/hudi/utilities/streamer/HoodieMultiTableStreamer.java

```diff
@@ -430,6 +432,99 @@ public static class Config implements Serializable {
 
     @Parameter(names = {"--help", "-h"}, help = true)
     public Boolean help = false;
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Do we need the equals and toString implementation here?

Reply:
I think that every config class should have equals, hashCode, toString for convenience. 
If not, can rollback them.

Reply:
Should we consider API like 
https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/builder/EqualsBuilder.html#reflectionEquals(java.lang.Object,%20java.lang.Object)?

The equals and hashCode are not implemented in other config classes as well. But if we need it we can retain the changes.

Reply:
I refered [Config on HoodieStreamer](https://github.com/apache/hudi/blob/master/hudi-utilities/src/main/java/org/apache/hudi/utilities/streamer/HoodieStreamer.java#L222)
rollback this, because it is not main for this PR